app/views/styleguide/pages/locator_entry.erb

```diff
@@ -13,7 +13,9 @@
 
         <p>Calls cost the same as a normal call - if your calls are free, it’s included.</p>
 
-        <p>If you can’t use the phone, you can book an appointment at your <a rel="external" href="https://www.citizensadvice.org.uk/">local Citizens Advice bureau</a>. </p>
+        <h2 id="who-can-book">Face-to-face appointments</h2>
+
+        <p>Check where you can have a face-to-face appointments near you.</p>
```

Comment:
"Check where you can have a face-to-face appointment near you." or "Check where you can have face-to-face appointments near you."